api/src/main/java/au/org/aodn/nrmn/restapi/controller/StagedJobController.java

```diff
@@ -378,4 +381,25 @@ public void downloadFile(final HttpServletResponse response, @PathVariable Strin
                     jobId, clientException.getMessage())));
         }
     }
+
```

Comment:
Bad function call that modify pass in object value, please return something instead of modify the existing one

Collection<Long> filterNonBottomDuplicateRows(Collection<Long>..... , Long);

Reply:
Got it. Will modify soon.